<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainUI</class>
 <widget class="QWidget" name="MainUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>387</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微軟正黑體</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>新楓之谷ARC計算機 v0.2    by 曉亦夏風</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/ARC1.png</normaloff>:/images/ARC1.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>0.920000000000000</double>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QLabel" name="arcimg1">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>60</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/images/ARC1.png</pixmap>
   </property>
  </widget>
  <widget class="QTextEdit" name="background">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>541</width>
     <height>391</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="acceptRichText">
    <bool>false</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="arcimg2">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>100</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/images/ARC2.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="arcimg3">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>140</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/images/ARC3.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="arcimg6">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>260</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/images/ARC6.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="arcimg5">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>220</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/images/ARC5.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="arcimg4">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>180</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/images/ARC4.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>61</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Lv.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>100</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Lv.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>140</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Lv.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>260</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Lv.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>181</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Lv.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>220</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Lv.</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Arc1LV">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>61</y>
     <width>25</width>
     <height>31</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="toolTip">
    <string>0~20</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="Arc2LV">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>25</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>0~20</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="Arc3LV">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>25</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>0~20</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="Arc6LV">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>260</y>
     <width>25</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>0~20</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="Arc5LV">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>220</y>
     <width>25</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>0~20</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="Arc4LV">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>181</y>
     <width>25</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>0~20</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>20</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ARC類型：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="ArcMode">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>一般職業(主屬)、傑諾(三屬)、惡復(HP)</string>
   </property>
   <item>
    <property name="text">
     <string>一般職業</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>傑諾</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>惡復</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="Arc1current">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>60</y>
     <width>34</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>0~372</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>3</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>60</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QLabel" name="Arc1upgrade">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>升級所需數量</string>
   </property>
   <property name="text">
    <string>?</string>
   </property>
  </widget>
  <widget class="QLabel" name="Arc2upgrade">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>升級所需數量</string>
   </property>
   <property name="text">
    <string>?</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Arc2current">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>100</y>
     <width>34</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>0~372</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>3</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>100</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Arc3current">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>140</y>
     <width>34</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>0~372</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>3</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>140</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QLabel" name="Arc3upgrade">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>升級所需數量</string>
   </property>
   <property name="text">
    <string>?</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Arc4current">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>180</y>
     <width>34</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>0~372</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>3</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>180</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QLabel" name="Arc4upgrade">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>升級所需數量</string>
   </property>
   <property name="text">
    <string>?</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Arc5current">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>220</y>
     <width>34</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>0~372</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>3</number>
   </property>
  </widget>
  <widget class="QLabel" name="Arc5upgrade">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>220</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>升級所需數量</string>
   </property>
   <property name="text">
    <string>?</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Arc6current">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>260</y>
     <width>34</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>0~372</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>3</number>
   </property>
  </widget>
  <widget class="QLabel" name="Arc6upgrade">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>260</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>升級所需數量</string>
   </property>
   <property name="text">
    <string>?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>220</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>260</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>300</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>目前ARC：</string>
   </property>
  </widget>
  <widget class="QLabel" name="ArcTotal">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>300</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>340</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>屬性增加量：</string>
   </property>
  </widget>
  <widget class="QLabel" name="ApTotal">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>340</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>0</y>
     <width>20</width>
     <height>391</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>261</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>強化所需楓幣</string>
   </property>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Lv.</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ArcLV_from">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>30</y>
      <width>25</width>
      <height>31</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="toolTip">
     <string>1~19</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>65</x>
      <y>30</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>→</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Lv.</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ArcLV_to">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>30</y>
      <width>25</width>
      <height>31</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="toolTip">
     <string>2~20</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>$：</string>
    </property>
   </widget>
   <widget class="QLabel" name="cost">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>30</y>
      <width>47</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>99999</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>萬</string>
    </property>
   </widget>
  </widget>
  <zorder>background</zorder>
  <zorder>arcimg1</zorder>
  <zorder>arcimg2</zorder>
  <zorder>arcimg3</zorder>
  <zorder>arcimg6</zorder>
  <zorder>arcimg5</zorder>
  <zorder>arcimg4</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>Arc1LV</zorder>
  <zorder>Arc2LV</zorder>
  <zorder>Arc3LV</zorder>
  <zorder>Arc6LV</zorder>
  <zorder>Arc5LV</zorder>
  <zorder>Arc4LV</zorder>
  <zorder>label_7</zorder>
  <zorder>ArcMode</zorder>
  <zorder>Arc1current</zorder>
  <zorder>label_8</zorder>
  <zorder>Arc1upgrade</zorder>
  <zorder>Arc2upgrade</zorder>
  <zorder>Arc2current</zorder>
  <zorder>label_9</zorder>
  <zorder>Arc3current</zorder>
  <zorder>label_10</zorder>
  <zorder>Arc3upgrade</zorder>
  <zorder>Arc4current</zorder>
  <zorder>label_11</zorder>
  <zorder>Arc4upgrade</zorder>
  <zorder>Arc5current</zorder>
  <zorder>Arc5upgrade</zorder>
  <zorder>Arc6current</zorder>
  <zorder>Arc6upgrade</zorder>
  <zorder>label_12</zorder>
  <zorder>label_13</zorder>
  <zorder>label_14</zorder>
  <zorder>ArcTotal</zorder>
  <zorder>label_15</zorder>
  <zorder>ApTotal</zorder>
  <zorder>line</zorder>
  <zorder>groupBox</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
